src/js/common/schemaform/FormApp.jsx

```diff
@@ -25,7 +25,9 @@ export default class FormApp extends React.Component {
   }
 
   onbeforeunload = e => {
-    const endpoint = this.props.currentLocation.pathname.split('/').pop();
+    const { currentLocation } = this.props;
+    const trimmedPathname = currentLocation.pathname.replace(/\/$/, '');
+    const endpoint = trimmedPathname.split('/').pop();
```

Comment:
Should we just change things to do the `endsWith` check, instead of splitting on `/`? Looks like it would mean an update in two places and then we'd be consistent across these two files.